packages/merge-on-green/test/merge-on-green.ts

```diff
@@ -151,18 +152,10 @@ describe('merge-on-green', () => {
   const loggerStub = sandbox.stub(logger, 'error').throwsArg(0);
 
   beforeEach(() => {
-    probot = new Probot({
-      // eslint-disable-next-line node/no-extraneous-require
-      Octokit: require('@octokit/rest').Octokit,
+    probot = createProbot({
+      githubToken: 'abc123',
```

Comment:
I think in test we should get in the habit of providing an octokit constructor, like this:

https://github.com/googleapis/repo-automation-bots/blob/master/packages/release-please/test/release-please.ts#L53

Otherwise in production you might get a slightly different API surface than what probot provides by default.

You don't need to add this dependency, as it will come in by default with `gcf-util`.